tensorflow_serving/tools/docker/Dockerfile.devel

```diff
@@ -34,7 +34,7 @@ RUN pip install mock grpcio
 
 ENV BAZELRC /root/.bazelrc
 # Install the most recent bazel release.
-ENV BAZEL_VERSION 0.5.1
+ENV BAZEL_VERSION 0.5.4
```

Comment:
Thanks! Could you please update the version everywhere else as well:
https://github.com/tensorflow/serving/blob/828786c37a1f79bc1234a829efa1f0ca6d62da19/WORKSPACE#L28
tensorflow_serving/tools/docker/Dockerfile.devel-gpu
tensorflow_serving/g3doc/setup.md

Reply:
No problem - on it :)

Reply:
Done! Please check again. Should be all on bazel 0.5.4 now.  (Just one quick note - I notice the [most recent Bazel GitHub release](https://github.com/bazelbuild/bazel/releases) is actually 0.6.1. But as 0.5.4 is the minimal required from what I encountered and worked, I deliberately use 0.5.4 for now. This I guess will also avoid too many moving parts. My suggestion is to do this in phases. Get this all synced up to bazel 0.5.4 for the time being - knowing that it would work. At a later time we may upgrade to 0.6.x - when needed. (just to be safe / avoid too many moving parts)